### DISCUSSION TITLE
Preview my file

### DISCUSSION THREAD

**Discussion by Ferritin17** [community/community#139530] [New to GitHub]
Hi! I am totally new to GitHub. I am collaborating on a project with my professor and am using GitHub desktop to work on a local  repository, and am pushing/syncing my changes to the original repository online for him to check every other week. Here’s my question: I know how to print the web version of my book to see online, and how to use the codechat preview in the online codespace to preview how the changes I’m making look. How do I do that on GitHub desktop so I don’t have to sync to the original repository, and go there and view my changes? Thanks so much!!!

**Comment by VadymMakohon** [comment]
Hello, @Ferritin17!

To preview your changes locally on GitHub Desktop without syncing every time, here’s what i would do:

Try to use a Local server or life server for exampale in VSCode to see your changes without pushing them to github. Also it might be helpfull to use GitHub Codespaces. They can offer a live environment where you can preview your changes. If you have access to it, you can work directly in Codespaces to see your changes in real time.

If this response was helpful, I’d appreciate it if you could mark it as the answer. Thanks!
